client/src/components/DataFiles/DataFilesModals/DataFilesCompressModal.jsx

```diff
@@ -108,7 +108,7 @@ const DataFilesCompressModal = () => {
                 />
                 <p>
                   A job to compress these files will be submitted. The
-                  compressed file archive will appear in this directory.
+                  compressed file archive will appear in the Root directory.
```

Comment:
The behavior of the compress files util should be such that the output is stored in the directory the original file lives, not the root directory

Reply:
Now that I corrected `defaultPrivateSystem`, this now works as it did before.